zstacklib/zstacklib/utils/pci.py

```
@@ -1,6 +1,8 @@
 import os
 
+from zstackctl.zstackctl.reset_mini import bash_roe
```

Ruff (0.8.2)

`zstackctl.zstackctl.reset_mini.bash_roe` imported but unused

Remove unused import: `zstackctl.zstackctl.reset_mini.bash_roe`

(F401)